mypy/errors.py

```diff
@@ -761,6 +791,44 @@ def remove_duplicates(self, errors: List[ErrorTuple]) -> List[ErrorTuple]:
             i += 1
         return res
 
+    def save_baseline(self):
+        """Create/update a file that stores all errors"""
+
+        p = Path(".mypy/baseline.json")
+        if not p.parent.exists():
+            p.parent.mkdir()
+        json.dump(
+            filter_prefix(self.error_info_map),
+            p.open("w"),
+            default=lambda o: {".class": (t := type(o)).__module__ + "." + t.__qualname__} | o.__dict__
+        )
+
+    def load_baseline(self) -> None:
+        """Load baseline errors from baseline file"""
+
+        p = Path(".mypy/baseline.json")
```

Comment:
i'd suggest specifying the path somewhere else  instead of hardcoding it twice
```suggestion
    baseline_path = ".mypy/baseline.json"

    def save_baseline(self):
        """Create/update a file that stores all errors"""

        p = Path(self.baseline_path)
        if not p.parent.exists():
            p.parent.mkdir()
        json.dump(
            filter_prefix(self.error_info_map),
            p.open("w"),
            default=lambda o: {".class": (t := type(o)).__module__ + "." + t.__qualname__} | o.__dict__
        )

    def load_baseline(self) -> None:
        """Load baseline errors from baseline file"""

        p = Path(self.baseline_path)
```
or maybe u could specify it in a flag?

Reply:
want to take another look?